source/blog/2022-02-03-add-venmo-to-your-solidus-store.html.markdown

```diff
@@ -0,0 +1,153 @@
+---
+
+PayPal has contributed to Solidus massively over the last few years, helping us revitalize both `solidus_paypal_commerce_platform` and `solidus_paypal_braintree` extensions. Entering into the next PayPal phase that we promised, we are excited that both extensions now support Venmo!
+
+This latest addition to their SDK and Braintree allows another streamlined payment option for your customers at no additional cost.
+
+Having multiple finance options is vital as it allows for a higher conversion rate. As well as this, Venmo offers some unique features.
+
+### What is Venmo?
+Venmo allows customers to make payments in a mobile app effortlessly, and uniquely, the buyer has the <a href="https://help.venmo.com/hc/en-us/articles/217532327-Splitting-Sharing-Purchases" target="_blank" rel=”noreferrer”>option to split or share the purchase</a>.
+
+It is currently available to only US PayPal merchants and buyers. However, with its recent rise, **every 1 in 5** US citizens already use it each year.
+
+If you meet the <a href="https://developer.paypal.com/docs/business/checkout/pay-with-venmo/#eligibility" target="_blank" rel=”noreferrer”>eligibility requirements</a> you can integrate Venmo into your store where buyers will gain the following experiences:
+
+#### Mobile checkout:
+<ol>
+  <li>Buyer clicks the Venmo button and their mobile device switches to the Venmo app.</li>
+  <li>Buyer sees a payment review on the Venmo app and completes it.</li>
+  <li>Buyer gets switched back to your website to complete the payment.</li>
+</ol>
+
+<figure>
+  <img src="/blog/2022/02/03/add-venmo-to-your-solidus-store/venmo-mobile-flow.png"
+       alt="Venmo checkout process on mobile">
+  <figcaption>Venmo checkout process on mobile.</figcaption>
+</figure>
+
+#### Desktop checkout:
+<ol>
+  <li>Buyer clicks the Venmo button on the desktop to generate a QR code.</li>
+  <li>Buyer scans the QR code from the Venmo mobile app.</li>
+  <li>Buyer completes the payment review on the Venmo app.</li>
+  <li>Buyer completes the payment by the website on desktop.</li>
+</ol>
+
+<figure>
+  <img src="/blog/2022/02/03/add-venmo-to-your-solidus-store/venmo-desktop-flow.png"
+       alt="Venmo checkout process on desktop">
+  <figcaption>Venmo checkout process on desktop.</figcaption>
+</figure>
+
+*Simple!*
+
+### How to integrate Venmo?
+Assuming you have either the <a href="https://github.com/solidusio/solidus_paypal_braintree#installation" target="_blank">solidus\_paypal\_braintree</a> or <a href="https://github.com/solidusio-contrib/solidus_paypal_commerce_platform#installation" target="_blank">solidus\_paypal\_commerce\_platform</a> installed and setup, continue below. Otherwise, keep the update step and go straight to the integration. <a href="https://solidus.io/blog/2020/11/24/using-paypal-with-solidus.html" target="_blank">Unsure which to install?</a>
```

Comment:
It seems to me the last phrase should say "Otherwise, skip" and not "Otherwise, keep"